### ISSUE TITLE
How long does it take for the bot to accept the collaborator request?

### ISSUE THREAD

**Issue by cbillowes** [GoogleChromeLabs/lighthousebot#55]
I added lighthousebot as a collaborator and I have bee waiting for a response for close to 12 hours. How long does it take for the bot to accept the request? I would have thought it to be instant. Is this a manual process? Am I missing a step somewhere? 

I went to settings > collaborators > entered the bot name and clicked on add collaborator.

**Comment by Toolo** [comment]
Waiting for it to accept, too. I was expecting it to be automatic; if there's any process involved I'd be happy to create pull to update the README.md stating so 😃

**Comment by curiouscrusher** [comment]
I've been waiting for multiple collab requests for almost 2 weeks now, it'd be great to have some clarity on what the timeline/requirements is for collaboration requests to be accepted.

**Comment by hammerhernandez** [comment]
I have the same issue and I need to use it ASAP. 👍 

Can someone of Lighthouse explain it.

Best Regards, @ebidel 💯

**Comment by bzitzow** [comment]
+1
We are also waiting for lighthousebot to accept the request since May 5th. 
That's 20 days.
We just removed and added again but still waiting.

![image (5)](https://user-images.githubusercontent.com/473242/58374407-7a446980-7ef2-11e9-8a81-599aedb54aff.png)

**Comment by ebidel** [comment]
@addyosmani is someone looking after this?

**Comment by addyosmani** [comment]
@khempenius I'm trying to locate the Lighthousebot requests but don't seem to have had notifications about them. Have you received any?

**Comment by khempenius** [comment]
@addyosmani I have not received any requests.

**Comment by hammerhernandez** [comment]
Hi Guys,

Any resolution about it? Or something that we can do or help?

Best Regards, @ebidel  @addyosmani @khempenius

**Comment by sumit-gupta91** [comment]
I added lighthousebot as a collaborator and I have bee waiting for a response for close to 12 hours. How long does it take for the bot to accept the request? @ebidel @addyosmani

**Comment by ebidel** [comment]
Did you receive an email with an API key? It looks like the form is failing
to send out emails in some cases, although I just tried it myself and it
went through. If you don't receive an email try to fill it out again.

As for the bot accepting the collaborative GitHub request I don't have
access to those emails anymore. Someone at Google will need to log in to
the bot account and accept the requests.

**Comment by bzitzow** [comment]
@ebidel @addyosmani We are excited to have lighthouse checks as part of our CI pipeline. Getting the lighthousebot collaborator is the last step.

Any ideas on how we can reach out to "someone at google" so they can help get these requests approved?

**Comment by sumit-gupta91** [comment]
@ebidel Hi, yes I did receive an email as an API key, but lighthouse bot has yes been added as a collaborator it is waiting for a response from lighthouse bot.

**Comment by rayriffy** [comment]
+1

**Comment by ebidel** [comment]
Ok. Someone with credential access to the lighthouse bot account will need to approve the requests coming in. Unfortunately, I'm not sure got that completely figured out after I left the team. We were working with GH support to verify @addyosmani and @khempenius 's emails for the account.

**Comment by AngelGenchev95** [comment]
Hey @addyosmani @khempenius  has this been resolved? I sent a collaborator invite a couple of hours ago but I'm not sure when to expect for it to go through.

**Comment by addyosmani** [comment]
@ebidel When you get a chance, could you share (email) where you were getting these approval requests sent through to? Unfortunately even with access to the lighthousebot GH account, we're still not receiving these requests.

Folks: Katie and I discussed it. If we're unable to get Lighthousebot sorted in the coming weeks, we're going to accelerate getting together working examples of LightWallet (official perf budgeting in Lighthouse) working in CI and a codelab so you can still get the perf budgeting portion of this workflow at least.

**Comment by addyosmani** [comment]
Can someone try sending through new collaborator requests again so we can test this works now? :)

@khempenius I've gone ahead and:

1. Created a new email address for Lighthousebot notifications (lighthousebotofficial at gmail, you should have access)
2. Added this as the primary account notification email for `lighthousebot`
3. Verified email notifications should be coming through correctly here now

Although I have no way of receiving old lighthousebot approval requests, we should now be able to receive them for any new users. Keeping fingers crossed at least...

**Comment by sumit-gupta91** [comment]
Sending a new request right away

**Comment by addyosmani** [comment]
I've approved 7 requests that came through last night. Feel free to keep them coming. I've confirmed with at least one user everything is working as expected now.

To close this bug report up: the collaborator request currently requires manual approval. Katie and I will do our best to process these every few days. As we work on more official CI solutions, we'll be looking at how to automate everything to drastically reduce time between requests and approvals.
